stable/drone/templates/deployment-agent.yaml

```diff
@@ -54,6 +58,14 @@ spec:
           - name: {{ $key }}
             value: {{ $value | quote }}
           {{- end }}
+        livenessProbe:
+          httpGet:
+            path: "/healthz"
+            port: http
+        readinessProbe:
+          httpGet:
+            path: "/healthz"
+            port: http
```

Comment:
Can you make livenessProbe and readinessProbe optional (.Values.livenessProbe.enabled) and add customizable parameters? Take as an example the [etc-operator](https://github.com/helm/charts/blob/cbd5e811a44c7bac6226b019f1d1810ef5ee45fa/stable/etcd-operator/templates/operator-deployment.yaml). Thank you!

Reply:
Also, please set it to enabled by default with an initial dealy of 5 seconds.

Reply:
@christian-roggia We already discussed this in the thread above (having values for the sake of having values).

And why would you deviate from what is a Kubernetes default in a Helm chart? The `etcd-operator` you've reference has an `initialDelaySeconds` of 0.